docs/animations/Expressions.md

```diff
@@ -100,9 +100,9 @@ ExpressionAnimations can create some very powerful and unique experiences, but c
 
 ```csharp
 _parallaxExpression = compositor.CreateExpressionAnimation(
-	"(ScrollManipulation.Translation.Y + StartOffset) * ParallaxValue - 
-	(ScrollManipulation.Translation.Y + StartOffset)");
-``` 
+     "(ScrollManipulation.Translation.Y + StartOffset) * ParallaxValue - " +

```

Comment:
Looks like this was probably a bug to wrap the string across lines without an operator. Could also be pulled to one line, but guess it was originally wrapped so it wouldn't be too long